xml/art_sles_rpi_quick.xml

```diff
@@ -434,10 +434,10 @@
     <title>Writing the Image to the Card using <command>dd</command></title>
     <para>
      This command decompresses the image
-     <literal>SLES15-SP1-JeOS.aarch64-15.1-RaspberryPi-GM.raw.xz</literal> to the SD
+     <literal>SLES15-SP2-JeOS.aarch64-15.2-RaspberryPi-GM.raw.xz</literal> to the SD
```

Comment:
```suggestion
     <literal>SLES15-SP2-JeOS.aarch64-15.2-RaspberryPi-GM.raw.xz</literal> to the &microsd;
```

Reply:
We have 11 more occurrences of "SD card", which is not strictly wrong - the reason for &microsd; was the inconsistent spelling MicroSD vs. microSD. We could probably clean that up with more time on our hands.